app/assets/javascripts/answers.js

```diff
@@ -20,6 +20,12 @@ $(document).ready(function(e){
   $("#answers").hide();
   $("#response").hide();
 
+  if ($('#answer_text').val() == "") {$('#submit_answer').attr('disabled', true);}
+
+  $('#answer_text').keyup(function(){
+    $('#submit_answer').attr('disabled', !($('#answer_text').val()));
```

Comment:
If you are relying on jQuery to write your js then you should look into triggers: https://learn.jquery.com/events/introduction-to-custom-events/ is another best practice for factoring out code that gets re-used or triggered in different scenarios